dist/responsive_images.js

```diff
@@ -305,7 +305,11 @@ function intrinsicOrExtrinsicSizing( computedStyles ) {
 function getImgData( img ) {
 
   const imgData = imgFeatures( img );
-  imgData.url = img.currentSrc || img.src;
+  // Limit URLs to 200 chars so data URLs don't take up huge amounts of space.
```

Comment:
WDYT think @pmeenan @rviscomi @paulcalvano @max-ostapenko is 200 too much?

Reply:
200 should be fine but it might be better to put the elipsis in the middle and slice 100 off of each side so that the filename and extension can be kept (if there is one at the end).

Reply:
The other alternative is to record full URLs and just shortens data URLs. Could then be even stricter with first 10 chars or so.

@eeeps any thoughts here?

Reply:
I went with that approach instead.